### DISCUSSION TITLE
npm run start is not working in windows

### DISCUSSION THREAD

**Discussion by Jonty16117** [mkosir/express-typescript-typeorm-boilerplate#182] [Show and tell]
### Description

I have resolved it on my windows system using "npm install cross-env" package.

**Comment by mkosir** [comment]
Hey @Jonty16117 👋 
not sure exactly what you mean, are you refereeing solution for this [issue](https://github.com/mkosir/express-typescript-typeorm-boilerplate/issues/73)?
`npm run start` command is ment to be run on server/production, since it's not suitable for development (no debugging etc.), but nice to hear that it's running on Windows with `cross-env` package 👍

**Reply by Jonty16117** [reply]
Yes.... exactly
